### DISCUSSION TITLE
Terminal only installation

### DISCUSSION THREAD

**Discussion by mizzunet** [mmagnus/Pocket-Plus-Calibre-Plugin#44] [Ideas]
Can I setup this via terminal only? I'm thinking running crun job in a server I got. Is this possible? Thank you

**Comment by mmagnus** [comment]
Take a look here https://github.com/mmagnus/Pocket-Plus-Calibre-Plugin/discussions/39

I guess @alvaroreig could help you with that.

**Reply by rga5321** [reply]
@mizzunet it's pretty easy if you have access to a SMTP server and calibre installed in your server. Two lines of code:

> ebook-convert Pocket.recipe salida.mobi

> calibre-smtp --attachment salida.mobi --relay "smtp.domain.om" --port "yourport"  --username "user@domain.com" --password "yourpassword"  --encryption-method TLS --subject "doesn't really matter" "mailfrom@domain.com" "yourkindle@kindle.com"  "body doesnt matter either"


On first run you will see a link in the output that you have to click to generate authorization token. Once that is set, it works perfectly, although lately amazon makes me confirm the files via email: https://www.reddit.com/r/kindle/comments/dl58gz/amazon_emailing_me_to_verify_every_book_i_sent_to/
